src/analysis/inner_product_space/basic.lean

```diff
@@ -470,9 +483,12 @@ lemma dfinsupp.inner_sum {ι : Type*} [dec : decidable_eq ι] {α : ι → Type*
   ⟪x, l.sum f⟫ = l.sum (λ i a, ⟪x, f i a⟫) :=
 by simp [dfinsupp.sum, inner_sum] {contextual := tt}
 
-@[simp] lemma inner_zero_left {x : E} : ⟪0, x⟫ = 0 :=
```

Comment:
Does the linter complain about this one? I think it's work keeping it if possible just because it makes for a nicer `squeeze_simp` experience

Reply:
The linter is fine with the version in the PR, if you keep the `simp`-attribute for `inner_zero_left` it complains for good reasons.

Reply:
I bet if you swap the order of the lemmas (and do the proofs in an unnatural order) then it doesn't complain

Reply:
Are you sure? I had it once that the linter was telling me that a lemma could be proved with a lemma that was defined later.

Reply:
I've changed the priority, this makes it work just fine:
`example (x y : E) : ⟪-x, y⟫ = -⟪x, y⟫ := by squeeze_simp` now correctly outputs `Try this: simp only [inner_neg_left]`